src/query/procedure/mg_procedure_impl.cpp

```diff
@@ -2535,6 +2535,309 @@ mgp_error mgp_graph_get_vertex_by_id(mgp_graph *graph, mgp_vertex_id id, mgp_mem
       result);
 }
 
+mgp_error mgp_create_label_index(mgp_graph *graph, const char *label, int *result) {
+  *result = 1;
+  auto label_id = std::visit([label](auto *impl) { return impl->NameToLabel(label); }, graph->impl);
+  const auto index_res = std::visit(
+      memgraph::utils::Overloaded{
+          [label_id](memgraph::query::DbAccessor *impl) { return impl->CreateIndex(label_id); },
+          [label_id](memgraph::query::SubgraphDbAccessor *impl) { return impl->GetAccessor()->CreateIndex(label_id); }},
+      graph->impl);
+  if (index_res.HasError()) {
+    *result = 0;
```

Comment:
it doesn't make too much sense because then I have to deal with exception on a lower level and conclude the same thing hey it didn't succeed let's return false...